gems/pending/appliance_console/internal_database_configuration.rb

```diff
@@ -178,12 +178,8 @@ def create_postgres_root_user
     end
 
     def create_postgres_database
-      run_as_postgres("createdb -E utf8 -O #{username} #{database}")
-    end
-
-    # some overlap with PostgresAdmin
-    def run_as_postgres(cmd)
-      AwesomeSpawn.run("su", :params => {"-" => nil, nil => "postgres", "-c" => "#{PostgresAdmin.scl_enable_prefix} \"#{cmd}\""})
+      conn = PG.connect(:user => "postgres", :dbname => "postgres")
+      conn.exec("CREATE DATABASE #{database} OWNER #{username} ENCODING 'utf8'")
```

Comment:
@kbrock https://github.com/ManageIQ/manageiq-appliance/blob/master/TEMPLATE/var/opt/rh/rh-postgresql94/lib/pgsql/data/pg_ident.conf#L4 does that job. @matthewd had pointed me to that in an unrelated conversation so it was fresh in mind :smile: